### DISCUSSION TITLE
Github uploads widget.

### DISCUSSION THREAD

**Discussion by AlexSNIC** [community/community#108332] [Mobile] [answered]
Hello👋. I am trying to maintain a streak of GitHub uploads, however, sometimes my commits glitch and other times I just forget to upload my code😅... Could you create a widget for mobile that displays your commits just like in your profile? I, and probably many others would really appreciate it 😉

**Comment by HenSquared** [comment, marked as answer]
Hi @Alex-LikesPizza , 

This is supported on iOS: 

https://github.blog/changelog/2021-02-02-github-for-ios-contribution-graph-widget/#:~:text=GitHub%20for%20iOS%20v1.,keep%20up%20with%20your%20contributions.

However, I am assuming you are on Android. We have received similar feedback, so I will add your feedback request to an existing internal issue we have tracking that request. 

Thanks for using GitHub and for the feedback! :)

**Reply by AlexSNIC** [reply]
Thank you!
